src/vs/workbench/browser/workbench.ts

```diff
@@ -392,6 +393,9 @@ export class Workbench extends Layout {
 		// Register Commands
 		registerNotificationCommands(notificationsCenter, notificationsToasts);
 
+		// Register Notification Telemetry
+		this._register(instantiationService.createInstance(NotificationsTelemetryContribution));
```

Comment:
I would call this `NotificationsTelemetry` and just move it somewhere below the `// Instantiate Notification components` things above.